agents/labjack/labjack_agent.py

```diff
@@ -309,9 +347,10 @@ def start_acq(self, session, params=None):
 
                 # The labjack outputs at exactly the scan rate but doesn't
                 # generate timestamps. So create them here.
-                timestamps = [cur_time+i/scan_rate for i in range(scans_per_read)]
-                cur_time += scans_per_read/scan_rate
-                data['timestamps'] = timestamps
+                t1 = t0 + read_period
+                timestamps = np.linspace(t0, t1, scans_per_read)
+                data['timestamps'] = timestamps[downsample_mask]
+                t0 = t1
```

Comment:
I'm kind of skeptical of this method of generating timestamps, since there's nothing to protect against timestamp drifts if the sample rate isn't exactly what we expect. I didn't want to change the logic here in this PR, but maybe we should think about whether this is the correct way to calculate timestamps

Reply:
Agreed this is not good practice... unless the Labjack has access to a realtime clock, then it will drift relative to the agent's computer.  I think it should be fixed for real (unfortunately that's not trivial), but perhaps for now you could at least test for drift and write an error message.